src/components/Header/Header.tsx

```diff
@@ -1,48 +1,43 @@
 "use client";
 import NextLink from "next/link";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
-import { faBars, faUser, faXmark } from "@fortawesome/free-solid-svg-icons";
+import { faUser } from "@fortawesome/free-solid-svg-icons";
 import Link from "../Button-Link/Link/Link";
 import "./Header.css";
 import { useState } from "react";
-import Button from "../Button-Link/Button/Button";
 import { User } from "@/services/types";
+import ProfileSideNav from "../ProfileSideNav/ProfileSideNav";
 
 type Props = {
   nav?: { href: string; title: string }[];
   user?: User;
+  className?: string;
 };
 
-const Header = ({ nav, user }: Props) => {
-  const [showNav, setShowNav] = useState(false);
-
-  const displayNav = () => {
-    setShowNav(!showNav);
-  };
+const Header = ({ nav, user, className }: Props) => {
   return (
-    <header className="Header sticky bg-white top-0">
+    <header className={`Header ${className} z-10 sticky bg-white top-0`}>
       <div className="flex justify-between items-center p-4">
-        <Button
-          variant="text"
-          color="black"
-          onClick={displayNav}
-          type="button"
-          size="small"
-          className="menuButton"
-        >
-          <FontAwesomeIcon
-            icon={showNav ? faXmark : faBars}
-            className="h-5 w-5"
-          />
-        </Button>
         <NextLink href="/">
-          <img src="/assets/logo.png" alt="logo" width="80" className="logo" />
+          <img
+            src="/assets/logo.png"
+            alt="logo"
+            width="80"
+            height="80"
+            className="logo "
+          />
         </NextLink>
         {user !== undefined ? (
-          <NextLink href="/profile">
-            <FontAwesomeIcon icon={faUser} className="mr-2 h-5 w-5" />
+          <div className="user-profile relative cursor-pointer ">
+            <FontAwesomeIcon
+              icon={faUser}
+              className="mr-2 h-5 w-5 text-black"
+            />
             {user.fullName}
-          </NextLink>
+            <div className="p-0 md:p-2">
+              <ProfileSideNav className="absolute top-full right-0 hidden w-52 rounded shadow-md bg-white m-0" />
```

Comment:
Should `bg-white` be here or should the `ProfileSideNav` be in charge of it's background?

Answer: it should be in charge of it's background. Here we should only add classNames related to positioning.